kafka_consumer_freshness_tracker/src/main/java/com/tesla/data/consumer/freshness/Burrow.java

```diff
@@ -135,5 +140,9 @@ public ClusterClient(String cluster) {
     public String getCluster() {
       return cluster;
     }
+
+    public Map<String, Object> getClusterDetail() throws IOException {
```

Comment:
is this getting used somewhere?

Reply:
No not currently, was just following the pattern I saw with the other burrow requests above. I can remove

Reply:
Lets remove it then.

Reply:
removed